gems/pending/Gemfile

```diff
@@ -88,6 +89,8 @@ unless ENV['APPLIANCE']
     gem "phantomjs",     "=1.9.8.0",    :require => false
     gem "rspec",         "~>3.5.x",     :require => false
     gem "timecop",       "~>0.7.3",     :require => false
+    gem "vcr",           "~>2.6",       :require => false
+    gem "webmock",       "~>1.12",      :require => false
```

Comment:
But come to think of it, it's rather unclear which gems in the gems/pending/Gemfile are for just gems/pending code vs. ones that needed by gems/pending and the rails app.  I guess we should continue this convention until we decide otherwise... @Fryguy what do you think?